### PR TITLE
Fix node8 github action

### PR DESCRIPTION
**Pull request by atrovato**
Split of https://github.com/abandonware/noble/pull/171

### CODE REVIEW

**Comment by atrovato** [comment]
Requested from https://github.com/abandonware/noble/pull/171

**Comment by atrovato** [comment]
Build fails due to linter, as on master branch: https://github.com/abandonware/noble/runs/1768999112

File: .github/workflows/nodepackage.yml [hunk 1]
```diff
@@ -45,11 +45,17 @@ jobs:
...
+            # mocha 8+ is not compatible with Node 8
+            npm install mocha@7.2.0
+          fi
+
           npm run test
       - name: Lint
         run: |
-          if [ ${{ matrix.node }} = 6 ]; then
-            # ESLint 6+ is not compatible with Node 6
+          if [ ${{ matrix.node }} = 8 ]; then
```
Comment (rzr) [inline .github/workflows/nodepackage.yml:57]: keep node6 lines and add elif I will merge it

**Review by rzr — CHANGES_REQUESTED** [review]

File: .github/workflows/nodepackage.yml (hunk 1 above)
Comment (atrovato) [reply .github/workflows/nodepackage.yml:57]: Even if node 6 is not on github actions ci file?

Comment (rzr) [reply .github/workflows/nodepackage.yml:57]: it will preserve the legacy info and the comment wont be lost
but it's a not a big deal

Comment (atrovato) [reply .github/workflows/nodepackage.yml]: Done

**Review by rzr — APPROVED** [review]
maybe we should also pin OS releases

**Comment by atrovato** [comment]
I don't understand' what do you mean by "pin OS release"?

**Comment by rzr** [comment]
replace latest by current versions well it's not a big deal let's focus on open PR 1st :)

**Pull request merged**